<launch>

	<!-- Distance sensor placement -->
	<node pkg="tf2_ros" type="static_transform_publisher" name="distance_sensor_front_link" args="0.1 0.13 0.09 3.1416 0 0  /base_link /distance_sensor_front_link" />

	<node pkg="tf2_ros" type="static_transform_publisher" name="distance_sensor_back_link" args="-0.1 0.13 0.09 3.1416 0 0  /base_link /distance_sensor_back_link" />

	<node pkg="robp_boot_camp_distance_sensor" type="distance_sensor_node" name="distance_sensor_node">
		<remap from="/adc" to="/kobuki/adc" />
	</node>
</launch>